packages/story-editor/src/components/panels/design/sizePosition/sizePosition.js

```diff
@@ -256,6 +293,8 @@ function SizePositionPanel(props) {
         )}
         <Area area="x">
           <NumericInput
+            ref={highlightRef}
+            onKeyDown={handleCancelHighlight}
```

Comment:
There's a different pattern around the codebase that adds the event listener in the ref callback

like this: https://github.com/GoogleForCreators/web-stories-wp/blob/fb965df3af6bec607dc5e6dbb97856adcac1b2c8/packages/story-editor/src/components/panels/design/imageAccessibility/imageAccessibility.js#L75-L80

But react provides an api for doing this using its own event management. This does the same thing without needing to manually attach and remove event listeners from this component.